samples/openthread/coprocessor/README.rst

```diff
@@ -139,76 +140,104 @@ Testing
+
+      .. code-block:: console
+
+         leader_if => [
+           "NCP:State" => "associated"
+           "Daemon:Enabled" => true
+           "NCP:Version" => "OPENTHREAD/gde3f05d8; NONE; Jul  7 2020 10:04:51"
+           "Daemon:Version" => "0.08.00d (0.07.01-343-g3f10844; Jul  2 2020 09:07:40)"
+           "Config:NCP:DriverName" => "spinel"
+           "NCP:HardwareAddress" => [E8F947748F493141]
+           "NCP:Channel" => 26
+           "Network:NodeType" => "leader"
+           "Network:Name" => "My_OpenThread_network"
+           "Network:XPANID" => 0x048CA9024CD7D40F
+           "Network:PANID" => 0xDB92
+           "IPv6:MeshLocalAddress" => "fd04:8ca9:24c:0:ebb8:4ef3:d96:c4bd"
+           "IPv6:MeshLocalPrefix" => "fd04:8ca9:24c::/64"
+           "com.nestlabs.internal:Network:AllowingJoin" => false
+         ]
+
+      This output means that you have successfully formed the Thread network.
+
+      .. _ot_coprocessor_sample_testing_more_boards:
+
+   .. tab:: b. RCP
+
+      1. Connect the development kit's SEGGER J-Link USB port to the PC USB port with an USB cable.
+      #. Get the kit's serial port name (e.g. /dev/ttyACM0).
+      #. Connect OpenThread Daemon to the RCP OpenThread node on the device:
+
+         .. code-block:: console
+
+            sudo ./modules/lib/openthread/output/posix/bin/ot-daemon -v 'spinel+hdlc+uart:///dev/ttyACM0?uart-baudrate=1000000'
+
+      #. While an instance of OpenThread Daemon is running, use the built-in controller to manage the connected node:
 
-      wpanctl:leader_if> status
+         .. code-block:: console
 
-The final output will be similar to the following:
+            sudo ./modules/lib/output/posix/bin/ot-ctl
 
-.. code-block:: console
+         This opens the OpenThread command line, which you can use as normal.
 
-   leader_if => [
-     "NCP:State" => "associated"
-     "Daemon:Enabled" => true
-     "NCP:Version" => "OPENTHREAD/gde3f05d8; NONE; Jul  7 2020 10:04:51"
-     "Daemon:Version" => "0.08.00d (0.07.01-343-g3f10844; Jul  2 2020 09:07:40)"
-     "Config:NCP:DriverName" => "spinel"
-     "NCP:HardwareAddress" => [E8F947748F493141]
-     "NCP:Channel" => 26
-     "Network:NodeType" => "leader"
-     "Network:Name" => "My_OpenThread_network"
-     "Network:XPANID" => 0x048CA9024CD7D40F
-     "Network:PANID" => 0xDB92
-     "IPv6:MeshLocalAddress" => "fd04:8ca9:24c:0:ebb8:4ef3:d96:c4bd"
-     "IPv6:MeshLocalPrefix" => "fd04:8ca9:24c::/64"
-     "com.nestlabs.internal:Network:AllowingJoin" => false
-   ]
+         Alternatively, you can issue OpenThread CLI commands directly from the command line, using the ot-ctl binary:
 
-This output means that you have successfully formed the Thread network.
+         .. code-block:: console
 
-.. _ot_ncp_sample_testing_more_boards:
+            sudo ./modules/lib/output/posix/bin/ot-ctl state
 
-Testing network joining with more kits
---------------------------------------
+Testing NCP network joining with more kits
```

Comment:
As mentioned above we can achieve the same with the RCP as well.

Reply:
It may be enough to change the line below to two variants for NCP and RCP.
`wpantund -I *network_interface_name_kit2* -s *serial_port_name_kit2* -b *baudrate*`

```
NCP:
wpantund -I *network_interface_name_kit2* -s *serial_port_name_kit2* -b *baudrate*

RCP:
wpantund -s 'system:./output/posix/bin/ot-ncp spinel+hdlc+uart:///dev/ttyACM0?uart-baudrate=115200'
(or maybe to stay consistent:)
wpantund -I *network_interface_name_kit2* -s 'system:./output/posix/bin/ot-ncp spinel+hdlc+uart://*serial_port_name_kit2*?uart-baudrate=*baudrate*'

```